Formula/cpprestsdk.rb

```diff
@@ -33,18 +34,16 @@ def install
       #include <iostream>
       #include <cpprest/http_client.h>
       int main() {
-        web::http::client::http_client client(U("https://github.com/"));
+        web::http::client::http_client client(U("https://example.com/"));
         std::cout << client.request(web::http::methods::GET).get().extract_string().get() << std::endl;
       }
     EOS
-    flags = ["-stdlib=libc++", "-std=c++11", "-I#{include}",
-             "-I#{Formula["boost"].include}",
-             "-I#{Formula["openssl@1.1"].include}", "-L#{lib}",
-             "-L#{Formula["openssl@1.1"].lib}", "-L#{Formula["boost"].lib}",
-             "-lssl", "-lcrypto", "-lboost_random", "-lboost_chrono",
-             "-lboost_thread-mt", "-lboost_system-mt", "-lboost_regex",
-             "-lboost_filesystem", "-lcpprest"] + ENV.cflags.to_s.split
-    system ENV.cxx, "-o", "test_cpprest", "test.cc", *flags
-    system "./test_cpprest"
+    system ENV.cxx, "test.cc", "-std=c++11",
+                    "-I#{Formula["boost"].include}", "-I#{Formula["openssl@1.1"].include}", "-I#{include}",
+                    "-L#{Formula["boost"].lib}", "-L#{Formula["openssl@1.1"].lib}", "-L#{lib}",
+                    "-lssl", "-lcrypto", "-lboost_random", "-lboost_chrono", "-lboost_thread-mt",
+                    "-lboost_system-mt", "-lboost_regex", "-lboost_filesystem", "-lcpprest",
```

Comment:
```suggestion
                    "-lboost_system-mt", "-lboost_filesystem", "-lcpprest",
```
I think Boost.Regex is header-only these days, do this isn't needed anymore.

Reply:
Although a dylib exists for it, confirmed this isn't needed anymore. Also just changed out the rest of the single-threaded variants out for `-mt`, since why not.